configure.ac

```diff
@@ -131,8 +131,10 @@ AC_CHECK_FUNCS([bzero gettimeofday memmove memset mkdir strdup strrchr \
 LT_INIT
 AC_PROG_LIBTOOL
 
-# Decide if we're building on Windows early on.
 AM_CONDITIONAL([USING_WIN32], [test "x$host_os" = xmingw32])
+if test -z "${USING_WIN32_FALSE}"; then
```

Comment:
I expect so, stuff that is Windows code, as opposed to *nix, e.g. some of the networking stuff.